### DISCUSSION TITLE
Node modules error while updating umijs v3 to v4.

### DISCUSSION THREAD

**Discussion by Khadhiri786** [ant-design/ant-design#43434] [Q&A]
Error
![image](https://github.com/ant-design/ant-design/assets/60467237/26f74211-97d6-4350-a6c4-a1a347f33ca1)
![image](https://github.com/ant-design/ant-design/assets/60467237/f449eb47-4a05-477a-93da-c2df7419eb95)

package.json
antd version:5.2.3
react:18.2
umijs:4.0.72

**Comment by zombieJ** [comment]
remove babel-plugin-import since v5 no need to use less anymore.

**Reply by Khadhiri786** [reply]
@zombieJ  Removed babel-plugin-import. Now I'm getting this issue
![image](https://github.com/ant-design/ant-design/assets/60467237/f56cc4cc-7d20-4161-9b02-2739c1e00a21)

**Reply by yoyo837** [reply]
ref https://github.com/ant-design/ant-design/issues/39973

**Reply by Khadhiri786** [reply]
@yoyo837  I have added this line in config
`  antd: { import: true },
`
getting error like this
![image](https://github.com/ant-design/ant-design/assets/60467237/4f7c2835-f240-4deb-985e-0400a90e8b96)
